components/automate-ui/package.json

```diff
@@ -18,86 +18,88 @@
     "test": "ng test --watch=false --code-coverage --source-map=false --configuration=no_auth",
     "test:watch": "ng test --watch=true --browsers='Chrome' --source-map=true --configuration=no_auth",
     "e2e": "ng e2e",
-    "lint": "/bin/bash -c 'exec 5>&1; t1=$(ng lint | grep -v \": analyzing \" | grep -v \"select is deprecated\" | tee /dev/fd/5; exit ${PIPESTATUS[0]}); if [ $? -ne 0 ]; then exit 1; else echo $t1 | grep \"is deprecated\" >/dev/null; test $? -ne 0; fi'",
+    "lint": "ng lint",
```

Comment:
If you are curious, that kludge being removed was to eliminate lots of noise in lint output due to the fact that Angular first deprecated a variant of `select`... then changed their minds!